ipamutils/utils.go

```diff
@@ -31,16 +48,22 @@ type NetworkToSplit struct {
 	Size int    `json:"size"`
 }
 
-func init() {
```

Comment:
isn't `init` easier to reason with than `initDefaults` ?

Reply:
You mean "using an `init()` function"? Or to rename the `initDefaults` (sync.Once) to `init`? (not sure if the latter works; I suspect `init` is somewhat "reserved"))